son-gtklic/docs/swagger.yaml

```diff
@@ -0,0 +1,547 @@
+# this is an example of the Uber API
+# as a demonstration of an API spec in YAML
+swagger: '2.0'
+info:
+  title: Sonata's License Manager
+  description: Manage licenses Sonata services.
+  version: "1"
+# the domain of the service
+host: sonata-vm1.aws.atnog.av.it.pt
+# array of all schemes that your API supports
+schemes:
+  - http
+# will be prefixed to all paths
+basePath: /api/v1
```

Comment:
`X-Version` is a strong candidate for that, though we haven't decided it yet. Again, this LM module is going to be called from the API Gateway only, but it would be good to have the same mechanism for versioning (in this case). The idea is the module should send back that header in the response. We have some time to adjust this idea, but the mechanism can be settled now.